README.md

```diff
@@ -0,0 +1,12 @@
+# Pass arguments (e.g. change database location)
```

Comment:
no user will see this. FYI

Reply:
and it this doesn't work with what's just in the .desktop, then it should be fixed.